src/hooks/useSelectedNetwork.ts

```diff
@@ -32,6 +36,8 @@ export const useSelectedNetwork = (): {
 
       setLocalStorageNetwork(network);
       dispatch(setSelectedNetwork(network));
+      const chainId = Number(ENVIRONMENT_CONFIG_MAP[selectedNetwork].ethereumChainId);
```

Comment:
`chainId` is now used in quite a few files. It may be worth adding to a hook similar to `useTokenConfigs` or `useURLConfigs`.

note: can be a fast follow by the dYdX team

Reply:
i'll leave this one for now